### DISCUSSION TITLE
Unable to install addin

### DISCUSSION THREAD

**Discussion by dharanidharandharmasivam** [cake-build/cake#3087] [Q&A] [answered]
For my automation, in my cake file, I tried to install my addin from my local site, not from nuget.org. While running the PowerShell script I am facing the below issue.

`Error: Failed to install addin`

But when I tried to give the domain link with addin as below, it works in windows machine but the same not work with my mac machine and receiving the above error.

`#addin nuget:http://nexus.aaa.com/repository/nuget-hosted?package=Flutter.Build`

Here aaa denotes our local domain, this is not our exact, just for reference I have shared this.

I can use the package.config file to download my adding, but the version of my addin will change, so I can't use that.

Previously it was working fine. This issue occurs after updating the packages of Visual Studio 2019 and I am using the cake version 0.38.5.

Can you guys help me out on this to resolve?

**Comment by augustoproiete** [comment, marked as answer]
Here are a couple of initial steps you can try and see if you can troubleshoot it:

1. Run Cake with diagnostic verbosity and see if any more details about the error appear `dotnet cake --verbosity=diagnostic`
2. Try the latest version of the Cake runner ([1.0.0-rc0003](https://cakebuild.net/blog/2021/01/cake-v1.0.0-rc0003-released) as of this writing) to see if it could be something that has already been fixed

If the above doesn't give you any clues, post the log of (1) here to see if anyone can spot something.

Also, given that it works on one machine, but not on the other, is there a possibility it's network related? e.g. Firewall blocking your Mac from accessing the Nexus server, or needing to be connected to a company VPN, etc.
